tfjs-backend-webgl/src/backend_webgl.ts

```diff
@@ -623,18 +622,6 @@ export class MathBackendWebGL extends KernelBackend {
     return this.texData.get(dataId);
   }
 
-  private getCPUBackend(): KernelBackend|null {
-    if (!env().getBool('WEBGL_CPU_FORWARD')) {
-      return null;
-    }
-
-    if (this.cpuBackend == null) {
-      this.cpuBackend = engine().findBackend('cpu');
```

Comment:
Q: When to set `this.cpuBackend` since it is still used in below code? Otherwise, we may meet memory leak issue.
```
  numDataIds() {
    return this.texData.numDataIds() +
        (this.cpuBackend ? this.cpuBackend.numDataIds() : 0) -
        this.pendingDeletes;
  }

Reply:
Nice catch, thank you Jiajia. I just removed it in this PR: https://github.com/tensorflow/tfjs/pull/4920